### PR TITLE
update doc

### PR DESCRIPTION
**Pull request by alavieille**

### CODE REVIEW

File: en/developer_guide/entity_list_ajax_pagination.rst [hunk 1]
```diff
@@ -142,25 +142,29 @@ The ``descriptionEntity`` property linked to the ``Redirection`` will be:
...
-
-* ``name`` : name of the entity's property
+* ``field`` : name of the entity's property
 * ``type`` : type of the entity's property
 * ``key`` : name of the property in your facade used by the dataTable
 
-Open Orchestra provides the ``Search`` annotation to help you describe the mapping directly
-in your entities.
+This mapping can be specified in a number of different formats including YAML, XML or
+directly inside your entities class via the annotations
```
Comment (ngilain) [inline en/developer_guide/entity_list_ajax_pagination.rst]: via annotations

File: en/developer_guide/entity_list_ajax_pagination.rst [hunk 2]
```diff
@@ -200,15 +204,72 @@ For instance, the ``Redirection`` entity will look like:
...
+
+    <search-mapping>
+        <class name="AppBundle\Document\AppBundle\Document">
+            <field field="locale" key="locale" />
+            <field field="siteName" key="site_name" />
+            <field field="routePattern" key="route_pattern" />
+            <field field="url" key="redirection, url" />
+            <field field="locale" key="locale" />
+            <field field="permanent" key="permanent" type="boolean"/>
+            <field field="permanent" key="fake_property2, fake_property_multi"/>
```
Comment (ngilain) [inline en/developer_guide/entity_list_ajax_pagination.rst]: fake_property is a naming for test, in a doc you should use something more explicit like a concrete example

File: en/developer_guide/entity_list_ajax_pagination.rst [hunk 3]
```diff
@@ -200,15 +204,72 @@ For instance, the ``Redirection`` entity will look like:
...
+-------
+ Notes
+-------
+
+In mapping :
+
+* ``type`` will take ``string`` as a default parameter if it is not specified.
+* With the mapping in annotation ``name`` will take the name of the property if it is not set.
+
+By default the mapping in xml or yaml should be in folder ``Ressources/config/search`` of your bundle.
```
Comment (ngilain) [inline en/developer_guide/entity_list_ajax_pagination.rst]: "By default the xml and yaml mapping files should be located in the folder 'xxx' of you bundle"

File: en/developer_guide/entity_list_ajax_pagination.rst [hunk 4]
```diff
@@ -200,15 +204,72 @@ For instance, the ``Redirection`` entity will look like:
...
+To specify an other folder for the mapping files you must change the configuration :
```
Comment (ngilain) [inline en/developer_guide/entity_list_ajax_pagination.rst]: To specify another folder, you must [...]

**Comment by alavieille** [comment]
@itkg-ngilain @itkg-nanne

**Comment by ngilain** [comment]
:+1:

**Comment by veolys-nanne** [comment]
in Usage table, remove Name is the `name` of column

**Comment by veolys-nanne** [comment]
in Usage table replace Number of records that the table can display in the current draw   by Number of records displayed in the current draw

**Comment by veolys-nanne** [comment]
:+1: after usage table changes

**Pull request merged**